src/core/actors/Tribe.java

```diff
@@ -325,11 +325,18 @@ public void meetTribe(Random r, Tribe[] tribes, int tribeID) {
 
         boolean[] inMetTribes = new boolean[tribes.length];
         //loop through all tribes
+        for (int i = 0; i<inMetTribes.length; i++){
```

Comment:
This for loop is not necessary (new boolean[x] initializes the contents of the array to false by default).